weekly.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Weekly Project 1</title>
+  </head>
+  <body style="font-family: Arial, Helvetica, sans-serif; font-size: 16px; line-height: 20px;">
+    <h1>Lorem ipsum dolor sit</h1>
+    <p>
+      Lorem ipsum dolor sit amet <i>consectetur</i>. Enim sed mauris suspendisse duis
+      feugiat mauris <strong>consectetur</strong>sed tincidunt. Enim eleifend <br>feugiat fermentum
+      integer enim quisque nibh scelerisque. Cursus vitae aliquam lorem nisi
+      maecenas pellentesque. Vitae aliquet arcu <br>quis viverra. Iaculis massa
+      elementum adipiscing pulvinar vitae et turpis sollicitudin quis. Nisi
+      viverra turpis risus eget enim. Platea<br>aliquet consectetur viverra at
+      turpis lorem netus. <strong>Interdum</strong>id ultricies facilisis nisl ac lectus
+      vestibulum. Elementum ut quam amet quis. <br><br>
+      <strong>Read More</strong>
+    </p>
+    <img src="img/Shoe_1.jpg" width="230px">
+    <img src="img/Shoe_1.jpg" width="230px">
+    <img src="img/Shoe_1.jpg" width="230px">
+    <img src="img/Shoe_1.jpg" width="230px">
```

Comment:
Since all these are related, I think the best way to organize them is to put them together in a div or section element.